### DISCUSSION TITLE
Change hover color for all components

### DISCUSSION THREAD

**Discussion by wackazong** [saadeghi/daisyui#2959] [Q&A]
The default theme uses a hover color that is quite dark for my taste. I would like it to be something lighter. Is there an idiomatic way to accomplish this without breaking the themeability?

**Comment by saadeghi** [comment]
All components? Only `btn` changes the color to a darker shade when hovered. You can change the hover style using `hover:`

**Reply by wackazong** [reply]
Buttons, Select, etc.

Could you give an example on how to change that?

**Comment by saadeghi** [comment]
Select does not have a color change on hover.

For button, here's an example:   
https://play.tailwindcss.com/8I2vc5tjQn

Let me know if you have a question.
